### PR TITLE
Feat/revise search detail page

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/views/Search/AggregateResults/Tags.tsx [hunk 1]
```diff
@@ -24,57 +27,87 @@ const AggregateTagResults = () => {
...
-                  <TagDigest.Concise tag={node} showArticlesNum />
-                </Card>
-              </List.Item>
-            )
-        )}
-      </List>
+  // load next page
+  const loadMore = () => {
+    analytics.trackEvent('load_more', {
+      type: 'search_article',
```
Comment (robertu7) [inline src/views/Search/AggregateResults/Tags.tsx:61]: `search_tag`?

File: src/views/Search/AggregateResults/Users.tsx [hunk 2]
```diff
@@ -41,18 +44,51 @@ const AggregateUserResults = () => {
...
+          />
+        }
+      />
+    )
+  }
+
+  // load next page
+  const loadMore = () => {
+    analytics.trackEvent('load_more', {
+      type: 'search_article',
```
Comment (robertu7) [inline src/views/Search/AggregateResults/Users.tsx:63]: `search_user`?

File: src/views/Search/AggregateResults/index.tsx [hunk 3]
```diff
@@ -1,17 +1,66 @@
...
+        </span>
+        <span>
+          <span className="titleMiddle">&nbsp;{q}</span>
+        </span>
+        <span className="titleRight">
+          &nbsp;
+          <Translate zh_hans="的搜索結果" zh_hant="的檢索結果" en="" />
+        </span>
+      </section>
+      <Tabs>
```
Comment (robertu7) [inline src/views/Search/AggregateResults/index.tsx:49]: spacing isn't same as the design

image

image

File: src/components/Search/SearchBar/index.tsx [hunk 4]
```diff
@@ -94,6 +131,18 @@ export const SearchBar: React.FC<SearchBarProps> = ({
               />
 
               <SearchButton />
+              {search.length > 0 && (
+                <ClearButton
+                  onClick={() => {
```
Comment (robertu7) [inline src/components/Search/SearchBar/index.tsx:136]: can use Formik's `setValues` instead?

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: src/components/ArticleDigest/Concise/index.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+        {cover && (
+          <div className="cover">
+            <ResponsiveImage url={cover} size="144w" smUpSize="360w" />
+          </div>
+        )}
+      </section>
+
+      {/* <FooterActions article={article} inCard date={date} {...controls} /> */}
+      <section>
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/components/ArticleDigest/Concise/index.tsx]: Only show created time

**Review by robertu7 — APPROVED** [review]

**Pull request merged**